pkg/monitoring/metrics.go

```diff
@@ -26,10 +27,15 @@ var (
 			Name: "kubevirt_cnao_cr_ready",
 			Help: "Cnao CR Ready",
 		})
+	kmpDeployedGauge = prometheus.NewGauge(
```

Comment:
kmp prefix is needed ? not sure

Reply:
what do you mean?

Reply:
i confused something, nevermind, its fine